src/context/loadCustomVolumes.tsx

```diff
@@ -20,7 +20,7 @@ export const loadCustomVolumes = async (
       const isCustom = volume.type === "custom";
 
       if (isCustom && isFilesystemOrBlock) {
-        result.push({ ...volume, pool: pool.name });
+        result.push({ ...volume, pool: pool.name, driver: pool.driver });
```

Comment:
I think we don't need the driver here, do we?

Reply:
right this was put in place for filtering ceph volumes when attaching disk device to instances. It's no longer needed will remove.